.github/workflows/cont_integration.yml

```diff
@@ -24,7 +28,7 @@ jobs:
       - name: Install rustup
         run: curl https://sh.rustup.rs -sSf | sh -s -- -y
       - name: Set default toolchain
```

Comment:
Maybe change this to "MSRV toolchain"?

Reply:
hmm, but it also gonna use the stable one due to the matrix strategy 🤔

Reply:
This sets the "default" toolchain (stable or 1.63.0) for each test run so I think the name is fine.